<?xml version="1.0"?>
<launch>
  <node name="robot_navigation" pkg="robot_navigation" respawn="false" type="robot_navigation_node" output="screen">
    <param name="local_planner" value="mesh_navigation/mesh_local_planner" />
    <param name="global_planner" value="mesh_navigation/mesh_global_planner" />
    <param name="global_frame" value="/map" />
    <param name="robot_frame" value="/base_footprint" />
    <param name="mesh_global_planner/edge_growing_angle" value="0.52359877559" />
    <param name="mesh_global_planner/edge_growing_dist" value="0.2" />
    <param name="mesh_global_planner/graph_base_type" value="vertex_graph" />
  </node>
</launch>
